### PR TITLE
reasonable message

### PR DESCRIPTION
**Pull request by salmaniqbal**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,6 +5,6 @@
   <title>Docker</title>
 </head>
 <body>
-  <h2>Hi, I am your friendly website</h2>
+  <h2>Next week we start at 8:30am</h2>
```
Comment (salmaniqbal) [inline index.html]: we will actually start at 9:30 so please update this

Comment (salmaniqbal) [reply index.html]: this is now fixed

**Review by salmaniqbal — COMMENTED** [review]
looks good

**Review by ThisIsRahmat — APPROVED** [review]
approved

**Pull request merged**